sci-biology/bx-python/bx-python-0.7.2.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1
+
+DESCRIPTION="Library for rapid implementation of genome scale analyses. "
+HOMEPAGE="https://bitbucket.org/james_taylor/bx-python/wiki/Home"
+SRC_URI="mirror://pypi/b/bx-python/bx-python-${PV}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+IUSE=""
+
+DEPEND="dev-python/cython[${PYTHON_USEDEP}]
+	    dev-python/numpy[${PYTHON_USEDEP}]"
+RDEPEND="dev-python/numpy[${PYTHON_USEDEP}]"
```

Comment:
RDEPEND="dev-python/numpy[${PYTHON_USEDEP}]"
    DEPEND="${RDEPEND}
        dev-python/cython[${PYTHON_USEDEP}]"

would be better.